src/ripple/json/tests/Object.test.cpp

```diff
@@ -221,8 +215,8 @@ class JsonObject_test : public TestOutputSuite
 #ifdef DEBUG
         setup ("repeating keys");
```

Comment:
Nope!  It's correct as it is.

There's error checking that's only performed if DEBUG is defined - because it's a tiny bit expensive.  This is code to test that that error checking works.